### PR TITLE
Add selector hasAttribute method (closes #1045)

### LINKED ISSUE #1045: Add 'hasAttribute' method to DomNodeState

### PR DESCRIPTION
**Pull request by inikulin**
\cc @helen-dikareva @AlexanderMoskovkin

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

File: src/client-functions/selectors/add-api.js [hunk 1]
```diff
@@ -168,6 +168,16 @@ function addSnapshotPropertyShorthands (obj, getSelector, customDOMProperties, c
         });
     };
 
+    obj.hasAttribute = attrName => {
+        var callsite = getCallsiteForMethod('getAttribute');
```
Comment (helen-dikareva) [inline src/client-functions/selectors/add-api.js]: `var callsite = getCallsiteForMethod('getAttribute'); ` -> `var callsite = getCallsiteForMethod('hasAttribute');`

Comment (inikulin) [inline src/client-functions/selectors/add-api.js]: Nice catch

**Comment by noderat** [comment]
It looks like this never made it into the documentation.

**Comment by helen-dikareva** [comment]
@noderat you can find it [here](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/dom-node-state.html#members-specific-to-element-nodes) in **Method** section

**Pull request merged**